rskj-core/src/test/java/org/ethereum/rpc/Simples/SimpleWorldManager.java

```diff
@@ -21,17 +21,17 @@
 import co.rsk.metrics.HashRateCalculator;
 import co.rsk.net.BlockProcessor;
 import co.rsk.net.simples.SimpleBlockProcessor;
+import org.ethereum.core.Blockchain;
 import org.ethereum.core.PendingState;
 import org.ethereum.db.BlockStore;
 import org.ethereum.listener.CompositeEthereumListener;
 import org.ethereum.listener.EthereumListener;
-import org.ethereum.manager.WorldManager;
 import org.ethereum.net.client.ConfigCapabilities;
 
 /**
  * Created by Ruben Altman on 09/06/2016.
  */
-public class SimpleWorldManager implements WorldManager {
+public class SimpleWorldManager {
```

Comment:
Removing SimpleWorldManager should be much easier now, it should be added to this PR